develop/webservices/WS_v1/WS_Methods_v1/AddCommentToAlarmV2.md

```diff
@@ -0,0 +1,23 @@
+---
+uid: AddCommentToAlarmV2
+---
+
+# AddCommentToAlarmV2
+
+Use this method to add a comment to an alarm.
+
+Available from DataMiner 10.4.11 onwards.
+
+## Input
+
+| Item        | Format  | Description |
+|-------------|---------|-------------|
+| connection  | String  | The connection ID. See [ConnectApp](xref:ConnectApp). |
+| dmaID       | Integer | The DataMiner Agent ID. |
```

Comment:
should we clarify this is the dataminer agent id of the element and not the agent currently hosting the element?

Reply:
(same for the other below)